README.md

```diff
@@ -123,9 +123,114 @@ yarn seed
   yarn start
 ```
 
+
+## Deployment Guide 🚀
+
+Follow these steps to deploy the Chat-App to a production environment:
+
+### 1. Setting Up the Server 🌐
+
+- Choose a cloud provider like **AWS**, **DigitalOcean**, or **Heroku** to host the backend server.
+- Ensure that **Node.js** and **Yarn** are installed on the server.
+- Make sure **MongoDB** is accessible—either by using a cloud MongoDB provider like **MongoDB Atlas** or setting up a local MongoDB instance.
+
+### 2. Configure Environment Variables 📋
+
+- Create a `.env` file in the backend directory.
+- Make sure to include the following variables:
+  
+  ```plaintext
+  MONGO_URI=your_mongodb_connection_string
+  JWT_SECRET=your_jwt_secret
+  NODE_ENV=production
+  PORT=your_preferred_port
```

Comment:
Mention there is a env.sample in the file to rename it to .env just tell all the above are there in .env.sample file and mention the above and just rename it as instructions in the readme